src/components/body/Body.ts

```diff
@@ -180,8 +180,12 @@ export class DataTableBody implements OnInit, OnDestroy {
   }
 
   rowClicked(event, index, row): void {
-    this.onRowClick.emit({event, row});
-    this.selectRow(event, index, row);
+    let options = { cancel: false };
```

Comment:
You are resetting the options object in the listener of row click? Thats interesting but I'm scared of race cases with that.